@navikt/core/react/src/overlays/floating-menu/Menu.tsx

```diff
@@ -543,38 +468,12 @@ const MenuItem = forwardRef<MenuItemElement, MenuItemProps>(
           // prevent Firefox from getting stuck in text selection mode when the menu closes.
           if (!isPointerDownRef.current) event.currentTarget?.click();
         })}
-        /* TODO: refactor these two, use correct onKeyDown/onKeyUp compose */
-        onKeyDown={composeEventHandlers(onKeyDown, (event) => {
-          if (disabled || event.repeat) {
-            return;
-          }
-          if (["Enter"].includes(event.key)) {
-            event.currentTarget.click();
-            /**
-             * We prevent default browser behaviour for selection keys as they should only trigger
-             * selection.
-             * - Prevents space from scrolling the page.
-             * - If keydown causes focus to move, prevents keydown from firing on the new target.
-             */
-            event.preventDefault();
-          }
-        })}
-        onKeyUp={composeEventHandlers(onKeyDown, (event) => {
-          if (disabled || event.repeat) {
-            return;
-          }
-          /* TODO: Extract keys to constants? */
-          if ([" "].includes(event.key)) {
-            event.currentTarget.click();
-            /**
-             * We prevent default browser behaviour for selection keys as they should only trigger
-             * selection.
-             * - Prevents space from scrolling the page.
-             * - If keydown causes focus to move, prevents keydown from firing on the new target.
-             */
-            event.preventDefault();
-          }
-        })}
+        onKeyDown={composeEventHandlers(onKeyDown, (event) =>
+          handleKey(event, "Enter"),
+        )}
+        onKeyUp={composeEventHandlers(onKeyUp, (event) =>
+          handleKey(event, " "),
+        )}
```

Comment:
Since it supports OverridableComponent, this can happen where the tag is changed to "a"
https://github.com/navikt/aksel/blob/6cdc324151e97d961fe4a71f2daf8d169dff4afa/aksel.nav.no/website/pages/eksempler/actionmenu/links.tsx